packages/@tinacms/cli/src/codegen/index.ts

```diff
@@ -34,73 +34,49 @@ export const generateTypes = async (
   }
 ) => {
   if (options.verbose) logger.info('Generating types...')
-  try {
-    let docs = []
-    let fragDocs = []
 
-    // Load GQL queries from user queries folder
-    try {
-      if (!options.noSDK) {
-        docs = await loadDocuments(queryPathGlob, {
-          loaders: [new GraphQLFileLoader()],
-        })
-      }
-    } catch (e) {
-      let showErrorMessage = true
-      const message: string = e.message || ''
-      if (
-        message.includes(
```

Comment:
I'm not sure if this check is because this error in particular is just verbose, but I figured it would be better than not logging anything.

Reply:
Oh, I see the cypress tests are failing with that error:
> Unable to find any GraphQL type definitions for the following pointers:

Not sure if that test should be updated, or if there's context I'm missing

Reply:
Update: Added that check back, as we need it incase the user doesn't have a queries folder